### ISSUE TITLE
BIOS locked .. and CH341a, 

### ISSUE THREAD

**Issue by abodypa9** [zlfn/Razerblade2020-Hackintosh#1]
Thank you, I have the same problem as the BIOS locked.. I have CH341a, but I need an explanation.. Thanks

**Comment by zlfn** [comment]
I'm sorry,  I don't think there's much help I can't give because all the tips I referenced have been erased...
It will be more helpful to ask on reddit or another community.

**Comment by zlfn** [comment]
All I can remember is I flashed the DVMT pre-alloc value changed BIOS with the ch341a and SOP8 clips.

**Comment by abodypa9** [comment]
Yes, thanks :) Do you have a telegram? I have photos that I saved from this forum before it was deleted

**Comment by zlfn** [comment]
I don't have my Telegram ID, but I have a Discord ID. Kiroo#3247
